src/js/themes/hpe.js

```diff
@@ -465,20 +465,20 @@ export const hpe = deepFreeze({
       `,
     },
     extend: ({ disabled, theme }) => `
-      ${
-        !disabled &&
-        `:hover {
-        background-color: ${
-          theme.global.colors['background-contrast'][
-            theme.dark ? 'dark' : 'light'
-          ]
-        };
-      }`
-      }
-      font-weight: 500;
-      width: auto;
-      padding: ${theme.global.edgeSize.xsmall} ${theme.global.edgeSize.small};
-    `,
+    ${
+      !disabled &&
+      `:hover {
+      background-color: ${
+        theme.global.colors['background-contrast'][
+          theme.dark ? 'dark' : 'light'
+        ]
+      };
+    }`
+    }
+    font-weight: 500;
+    width: auto;
```

Comment:
Do we still want vertical pad when the checkbox is outside of the formfield?

Reply:
oh right yes let me put that back thank you forgot